Contributing.md

```diff
@@ -13,3 +13,9 @@ Respect the minimum deployment target. If you are adding code that uses new APIs
 ### Forwards compatibility
 
 Please do not write new code using deprecated APIs.
+
+### Maintainers
+
+AloeStackView is developed and maintained by Marli Oshlack (marli@oshlack.com), Fan Cox (fan.cox@airbnb.com), and Arthur Pang (arthur.pang@airbnb.com).
+
+Please feel free to tag @marlimox, @fancox, and @apang42 on PRs for code review.
```

Comment:
This may be better suited for a PR template, but it's fine to have this here too.

Reply:
I'd suggest not having this here; if you want this to happen automatically, you could use github's Code Owners feature and it'll auto-tag you.

Reply:
Agreed that it makes sense to add these to a PR template, but I think it’s nice to provide some points of contact here as well to round out a guide on contributing.

Reply:
TIL the Code Owners feature existed!

Reply:
Fair enough. I was thinking of this more as a way of introducing points of contact rather than as an ownership mechanism. I wanted people who read this file to come away with an understanding of “Ah these are the people I can expect to interact with when I go to make a change to this project”. I can remove this part though if you don’t feel it’s useful.

Reply:
If being individually tagged would help you, go for it - i generally feel that "any comment anywhere on the repo" should be sufficient to get maintainers' attention.

Reply:
Yeah makes sense. I’ll remove this section. I agree, I don’t think it’s really helping achieve what I thought it would.